README.md

```diff
@@ -82,17 +82,42 @@ You may also need to restart the kubelet daemon on your master node to pick up t
 systemctl restart kubelet.service
 ```
 
-### Configure IAMIdentityMapping Custom Resource Definitions
-
-In the `master` version of the AWS IAM Authenticator you can configure your users using one of three methods. The `mapUsers` and `mapRoles` as seen in the [Full Configuration Format](#full-configuration-format), using the new (alpha) [Kubernetes Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), or using an [EKS aws-auth ConfigMap](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html). The `--backend-mode` flag determines which of these methods is enabled and their order of precedence (first match wins). The CRD and ConfigMap allow the authenticator server to update its IAM identity mappings without restarting. See [Issues #79](https://github.com/kubernetes-sigs/aws-iam-authenticator/issues/79) for more details.
-
-To setup an `IAMIdentityMapping` CRD you'll first need to `apply` the CRD manifest:
+### 4. Create IAM role/user to kubernetes user/group mappings
+The default behavior of the server is to source mappings exclusively from the
+`mapUsers` and `mapRoles` fields of its configuration. See [Full Configuration
+Format](#full-configuration-format) below for details.
+
+Using the `--backend-mode` flag, you can configure the server to source
+mappings from two additional backends: an EKS-style ConfigMap
+(`--backend-mode=EKSConfigMap`) or `IAMIdentityMapping` custom resources
+(`--backend-mode=CRD`). The default backend, the server configuration ConfigMap
+that's mounted by the server pod, corresponds to
+`--backend-mode=MountedConfigMap`. You can pass a comma-separated list of these
+backends to have the server search them in order. For example, with
+`--backend-mode=EKSConfigMap,MountedConfigMap`, the server will search the
+EKS-style ConfigMap for a mapping then the server configuration ConfigMap if it
+doesn't find one. This is useful if you're migrating your mappings from one
+backend to another.
+
+#### `MountedConfigMap`
```

Comment:
I get why you're renaming this to include "ConfigMap", but it doesn't strictly have to be one. What do you think about `MountedFile`?  Does renaming the old `ConfigMap` to `EKSConfigMap` make it less confusing to call this one `MountedFile`?

Reply:
Good point, yes I like `MountedFile`, I've renamed MountedConfigMap to MountedFile.